examples/pennylane/3_Hydrogen_geometry_with_VQE/3_Hydrogen_geometry_with_VQE.ipynb

```diff
@@ -4,14 +4,14 @@
    "cell_type": "markdown",
```

Comment:
<p>Both filename and title would be more accurate as Hydrogen Molecule, not simply Hydrogen.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>